include/clang/Driver/Options.td

```diff
@@ -2621,6 +2621,14 @@ def fintegrated_as : Flag<["-"], "fintegrated-as">, Flags<[DriverOption]>,
 def fno_integrated_as : Flag<["-"], "fno-integrated-as">,
                         Flags<[CC1Option, DriverOption]>, Group<f_Group>,
                         HelpText<"Disable the integrated assembler">;
+
+def fcheckedc_null_ptr_checks : Flag<["-"], "fcheckedc-null-ptr-checks">,
+  Group<f_Group>, Flags<[CC1Option]>,
+  HelpText<"Enable runtime null ptr checks">;
```

Comment:
Nit: "Enable runtime nullptr checks". Similarly, at other places.

Reply:
Thanks. I wanted to distinguish general "null pointers" from the C++ "nulllptr" keyword. That's the reason for the space in between. I guess I can spell it out as "null pointer".